.github/workflows/snyk-scan.yaml

```diff
@@ -0,0 +1,16 @@
+name: Snyk Monitor
+on:
```

Comment:
Might be useful to add "workflow_dispatch:" also so the action can be run on demand?